azerothcore/azerothcore-wotlk
# fix(DB): Image of Commander Ameer Quests Start/End

Some quests were started/ended by two creatures. 
Source: https://wowwiki.fandom.com/wiki/Image_of_Commander_Ameer

### What's right: 

Netherstorm quests started by creature 20482:

- Neutral 15 [70] Ethereum Data
- Neutral 15 [70] Potential for Brain Damage = High
- Neutral 15 [70] S-A-B-O-T-A-G-E
- Neutral 15 [70] Delivering the Message
- Neutral 15 [70+] Nexus-King Salhadaar (Group)

### What's wrong:
Blade's Edge Mountains quests are started AND ended by creatures 20482 & 22919:

- Neutral 15 [70] Purging the Chambers of Bash'ir
- Neutral 15 [70] The Mark of the Nexus-King
- Neutral 15 [70] Stasis Chambers of the Mana-Tombs 
- Neutral 15 [70] Nexus-Prince Shaffar's Personal Chamber 

Neutral 15 [70] The Eye of Haramad is started by creature 20482 instead of 22919.






## CHANGES PROPOSED:

-  Delete queststarter for quests 10975, 10976, 10977, 10981 and creature 20482
-  Delete questender for quest 10975, 10976, 10977, 10981, 10982 and creature 20482
- Update queststarter for quest 10982 to 22919


## ISSUES ADDRESSED:

- Closes 




## TESTS PERFORMED:

- Imported SQL on Win 10 pro x64 
- Tested all quests succesfully ingame




## HOW TO TEST THE CHANGES:


- Do the quests 10975, 10976, 10977, 10981, 10982 and check that they only start/end at creature 22919


Image of Commander Ameer - Netherstorm
.go 4019.941650 2317.070068 114.749313 530

Image of Commander Ameer - Blade's Edge Mountains
.go 3864.034424 5976.675293 290.619293 530


## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
